packages/components/src/components/carousel/carousel-styles.ts

```diff
@@ -212,19 +228,36 @@ export const getComponentCss = (
         })),
         position: 'relative',
         justifyContent: isInfinitePagination ? 'flex-start' : 'center',
-        width: `calc(${paginationActiveBulletSize} + ${paginationBulletSize} * 4 + ${spacingStaticSmall} * 4)`, // Width for five bullets (one active + spacing)
-        left: 'calc(50% - 42px)',
+        width: paginationWidth,
+        left: `calc(50% - (${paginationWidth}) / 2)`,
```

Comment:
Brackets shouldn't be needed I think.
```suggestion
        left: `calc(50% - ${paginationWidth} / 2)`,
```